custom_dc/setup_gcp.md

```diff
@@ -3,5 +3,73 @@ layout: default
 title: System Setup
 nav_order: 1
 parent: Custom Data Commons
-published: false
+published: true
 ---
+
+## Overview
+
+Custom Data Commons is deployed in Google Cloud Platform (GCP). The follow
+manual presents how to install a custom Data Commons instance in an existing GCP
+project (with id `PROJECT_ID`).
+
+### Domain
+
+The installation can use an existing domain, or create a default domain
+`<PROJECT_ID>-datacommons.com`.
+
+### Steps
+
+1. From [Google Cloud Console](https://console.cloud.google.com/), open Cloud
+   Shell by clicking on the icon, like below:
+
+   ![fa](/assets/images/custom_dc/install_step_1.png){: width="600" }
+
+1. Set the environment variables `PROJECT_ID` and `CONTACT_EMAIL` in the
+   terminal:
+
+   ```bash
+   export PROJECT_ID=<project_id>
+   export CONTACT_EMAIL=<email_address>
+   ```
+
+   ![fa](/assets/images/custom_dc/install_step_2.png){: width="600" }
+
+   Note: If this step fails, please contact support+custom@datacommons.org with
+   the errors.
+
+1. Please run the following installation command inside the terminal. This may
+   take up to 20 minutes to complete.
+
+   ```bash
+   curl -fsSL https://raw.githubusercontent.com/datacommonsorg/website/master/scripts/install_custom_dc.sh -o install_custom_dc.sh && \
```

Comment:
Can you help me change this to https://raw.githubusercontent.com/datacommonsorg/website/custom-dc-v0.1.0/scripts/install_custom_dc.sh please? Thanks!

Reply:
Done